packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-development.md

```diff
@@ -1,116 +1,106 @@
 ---
-id: Product development guide
+id: Develop for accessibility
 section: accessibility
 ---
 import { AccessibilityDatalist } from './accessibility-datalist.js';
 
-## General questions to consider
+PatternFly provides accessible components, but **we can't guarantee that your products will be accessible**. In order to ensure that your product is accessible, you will need to take additional steps during development. For guidance on this development work, you can refer to these guidelines.
 
-To assess how accessible your applications or products are, consider these questions:
-- **Is it discoverable/perceivable by all users?** If you can see or click on it, then it should be discoverable and perceivable by all users. You should determine if you can easily navigate to it by keyboard or by screen reader.
-- **Is it interactable?** If you can get to it, it should be easy to actually use once you’ve focused on the element. You should be able to interact with it by keyboard (for example, if it’s a button, can you press **Enter** to initiate the action?). Also use a screen reader to ensure you can initiate the action.
-- **Is it understandable?** If the element can be found and interacted with, it should be clear what this action does. If it’s a button, you should determine if it has visible text that would be clear out of context of the page. If it doesn’t have visible text, you should check if it has an aria-label or accessible name.
+## Prioritizing accessibility during development
 
+As you develop your product, it can be helpful to keep accessibility top of mind, by asking yourself a few questions: 
 
-## What products should address
+- **Can all users discover and perceive elements?**: If you can see or click on it, then all users should be able to locate and navigate to it. This should include those that use a keyboard or other assistive technology, such as a screen reader.
 
-Keep in mind that while PatternFly provides accessible components, we can't guarantee that your products will be accessible. There are a few items outside the scope of PatternFly, so you should keep these items in mind to ensure accessibility in anything you build: 
-<br />
+- **Can all users interact with elements?**. It should be easy for users to use an element once it's in focus. They should be able to interact with elements and initiate actions via keyboard (like pressing **Enter** to initiate a button action) and screen readers.
```

Comment:
```suggestion
- **Can all users interact with elements?**. It should be easy for users to use an element once it's in focus. They should be able to interact with elements and initiate actions via keyboard (like pressing <kbd>Enter</kbd> to initiate a button action) and screen readers.
```

Reply:
Ahh I was working locally yesterday and didn't end up pushing, so that's probably what you saw! I just pushed though & I think both your syntax comments should be addressed 😀 @thatblindgeye